docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -6,6 +6,41 @@
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+.The file and native realms are now enabled unless explicitly disabled.
+[%collapsible]
+====
+*Details* +
+The file and native realms are now enabled unless explicitly disabled. If
+explicitly disabled, the file and native realms remain disabled at all times.
+
+Previously, the file and native realms had the following implicit behaviors:
+
+* If the file and native realms were not configured, they were implicitly disabled
+if any other realm was configured.
+
+* If no other realm was available because realms were not configured or
+explicitly disabled, the file and native realms were enabled, even if explicitly
+disabled.
+
+*Impact* +
+To explicilty disable the file or native realm, set the respective
+`file.<realm-name>.enabled` or `native.<realm-name>.enabled` setting to `false`
+under the `xpack.security.authc.realms` namespace in `elasticsearch.yml`.
+
+The following configuration example disables the native realm, named `realm1`,
+and the file realm, named `realm2`:
```

Comment:
I think we should deemphasize of the realm names here by just saying:
```suggestion
The following configuration example disables the native realm and the file realm.
```

This is because names are somewhat irrelevant here. Both realms are singletons, you can use any names for them, they will be disabled regardless and you cannot enable it under a different name.